fearless/Common/Model/ChainEcosystem.swift

```diff
@@ -0,0 +1,13 @@
+enum ChainEcosystem: String, Equatable {
+    case kusama
+    case polkadot
+
+    var isKusama: Bool {
```

Comment:
please add isPolkadot case, otherwise you'll need to check !isKusama for case you need polkadot